client/test/cypress/integration/submissionreview.cy.js

```diff
@@ -297,39 +312,67 @@ describe("Submissions Review", () => {
     cy.dataCy("overallCommentReply").first().contains("This is a modified overall comment reply.")
   })
 
-  it("should allow inline comments to be modified", () => {
-    cy.task("resetDb")
-    cy.login({ email: "applicationadministrator@ccrproject.dev" })
-    cy.visit("submission/review/100")
+  // it("should allow inline comments to be modified", () => {
+  //   cy.task("resetDb")
+  //   cy.login({ email: "applicationadministrator@ccrproject.dev" })
+  //   cy.visit("submission/review/100")
```

Comment:
This function should be un-commented.

Reply:
In order for this test to pass, it would require creation of a new inline comment via cypress, which we discussed the difficulty of doing. Inline comment creation is also not being tested in this spec, so we discussed commenting it out.

Reply:
My takeaway from our discussion is that seeded inline comments would fulfill the needs of this test and that creation isn't necessary. The name of this test is `should allow inline comments to be modified` which is undoubtedly pertinent and satisfies the need to ensure that 1 of the 4 comment types that can be modified can be done so successfully.

Reply:
The problem with seeded comments is that the comment authors are currently generated randomly, so there is no way to always point to a comment that can be modified. We discussed the difficulty in creating a new inline comments using cypress to get around this, and ultimately decided that because Inline comment creation is not being tested either, commenting it out was an option.

Reply:
It's possible to update the seeding so that you can guarantee you have a comment to work with. If you need help, I'd be willing to assist and pair program.

Reply:
I thought we decided to not modify the seeder file, but I can make those changes again so that the inline comments that are seeded have defined users.

Reply:
I apologize for leaving the incorrect impression. Yeah, updating the seeder would be one of the simpler approaches I think.